src/wcs2_0/src/test/java/org/geoserver/wcs2_0/WCSTestSupport.java

```diff
@@ -343,6 +347,15 @@ public void cleanCoverages() {
         }
     }
 
+    public static void assertXpathCoordinate(
```

Comment:
Good idea! 
This could be useful elsewhere too... what about pushing it up to GeoServerAbstractTestSupport, with a configurable tolerance? 1E-9 makes some sense for geographic coordinate tests ([mandatory XKCD](https://xkcd.com/2170/)) but it's too strict for projected ones.

Reply:
Sure good idea; note I am a little hampered testing locally util this is in okay.

Reply:
It fit better in ``GeoServerSystemTestSupport`` along side other DOM manipulation and assert methods.